app/authentication.js

```diff
@@ -20,19 +21,23 @@ module.exports = function (app) {
     // Remove the domain from the username
     var username = name.substring(0, name.lastIndexOf('@'))
     // Get the role for the user
-    getUserRoleByUsername(username).then(function (role) {
-      done(null, {
-        name: displayName,
-        username: username,
-        user_role: role.role,
-        nameID: nameID,
-        nameIDFormat: nameIDFormat
+    getUserByUsername(username).then(function (user) {
+      getUserRoleByUsername(username).then(function (role) {
```

Comment:
We could use the methods provided by the userRolesService for getting the user and role.

Reply:
Done